pkgs/by-name/dp/dpp/package.nix

```
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, zlib
+, openssl
+, libsodium
+, libopus
+}:
+
+stdenv.mkDerivation rec {
+  pname = "DPP";
+  version = "10.0.26";
+
+  src = fetchFromGitHub {
+    owner = "brainboxdotcc";
+    repo = pname;
```

```suggestion
  pname = "dpp";
  version = "10.0.26";

  src = fetchFromGitHub {
    owner = "brainboxdotcc";
    repo = "DPP";
```
the attr and pname should ideally match